cmd/api/notifications/notifications.go

```diff
@@ -0,0 +1,38 @@
+package notifications
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+type Ntfy struct {
+	baseURL string
+	enabled bool
+	client  *http.Client
+}
+
+func NewNtfy(enableNotifications bool, notificationsBaseURL string) *Ntfy {
+	return &Ntfy{
+		baseURL: notificationsBaseURL,
+		enabled: enableNotifications,
+		client:  &http.Client{},
+	}
+}
+
+func (ntf *Ntfy) BookCreated(ctx context.Context, title string, inventory int) error {
+	if ntf.enabled {
+		req, err := http.NewRequestWithContext(ctx, http.MethodPost, ntf.baseURL+"_New_book_created", strings.NewReader(fmt.Sprintf("New book created:\nTitle: %s\nInventory: %v", title, inventory))) //Ntfy SEEMS NOT TO ACEPT SLASHS OR DOTS AT TOPIC
```

Comment:
Please break this down into more lines. There is too much stuff happening in a single line.
Create an `url` and a `message` variable to store the intermediary values.